src/main/java/uk/gov/ons/ctp/integration/censusfieldsvc/service/impl/LauncherServiceImpl.java

```diff
@@ -80,16 +81,19 @@ public LaunchDetails getEqUrl(String userId, UUID caseId) throws FieldServiceExc
     try {
       encryptedPayload =
           eqLaunchService.getEqLaunchJwe(
-              Language.ENGLISH,
-              Source.FIELD_SERVICE,
-              Channel.FIELD,
-              caseDetails,
-              userId,
-              questionnaireIdDto.getQuestionnaireId(),
-              questionnaireIdDto.getFormType(),
-              accountServiceUrl,
-              accountServiceLogoutUrl,
-              appConfig.getKeystore());
+              EqLaunchData.builder()
```

Comment:
I would prefer this to be declared as a separate variable before the call to `getEqLaunchJwe` . To me it would be clearer.